clang/lib/Driver/ToolChains/HIP.cpp

```diff
@@ -112,8 +112,10 @@ void AMDGCN::constructHIPFatbinCommand(Compilation &C, const JobAction &JA,
   // for backward compatibility. For code object version 4 and greater, the
   // offload kind in bundle ID is 'hipv4'.
   std::string OffloadKind = "hip";
+  // bundle ID equals 'hip' is always right.
   if (getAMDGPUCodeObjectVersion(C.getDriver(), Args) >= 4)
-    OffloadKind = OffloadKind + "v4";
+    //OffloadKind = OffloadKind + "v4";
```

Comment:
Command `clang-offload-bundler -targets=host-x86_64-unknown-linux,hipv4-amdgcn-amd-amdhsa–` seems to be unable to execute normally. The error printed out is `hipErrorNoBinaryForGpu: Unable to find code object for all current devices!`. But it can be executed normally when the `targets` option is changed to `hip-amdgcn-amd-amdhsa`, I haven't figured out the reason yet.